pyRF24Mesh/pyRF24Mesh.cpp

```diff
@@ -87,7 +87,11 @@ BOOST_PYTHON_MODULE(RF24Mesh)
             //uint16_t renewAddress(uint32_t timeout=MESH_RENEWAL_TIMEOUT);
             .def("renewAddress", &RF24Mesh::renewAddress, getNodeID_overload(bp::args("timeout")))
             //bool releaseAddress();
-            .def("releaseAddress", &RF24Mesh::releaseAddress)
+            .def("releaseAddress", (bool(RF24Mesh::*)()) & RF24Mesh::releaseAddress)
+#ifndef MESH_NO_MASTER
+            //bool releaseAddress(uint16_t address);
+            .def("releaseAddress", (bool(RF24Mesh::*)(uint16_t)) & RF24Mesh::releaseAddress, (bp::args("address")))
+#endif
```

Comment:
I want to test this in the python wrapper (on a master and a child) to ensure I did this correctly. I know it compiles, but that doesn't indicate expected runtime behavior.

Reply:
This seems to work as expected on master nodes and child nodes. However, I'm noticing problems with the overloads for `getNodeId()` and `renewAddress()` in the python wrapper.

These py bindings also don't expose the `RF24Mesh::addrList` & friends. So py users have no way of checking the assigned addresses on master nodes.

Reply:
The bindings for `getNodeID()` and `renewAddress()` are fixed now.

I gave up on exposing `RF24Mesh::addrList` because boost.python needs to understand how to convert the returned `addrListStruct*` array into a python list (or tuple) of wrapped `addrListStruct` objects.

FYI, all of this is already done in pyrf24 package, and the `get/setAddress()` with the new overloaded `releaseAddress(uint16_t address)` can be used as a limited alternative. So, I'm not too concerned about perusing this further.